pytorch_toolkit/formula_recognition/tools/export.py

```diff
@@ -0,0 +1,149 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import argparse
+import math
+import subprocess
+
+import numpy as np
+import torch
+import torch.onnx
+from im2latex.data.vocab import START_TOKEN, read_vocab
+from im2latex.models.im2latex_model import Im2latexModel
+from tools.utils.common import (DECODER_INPUTS, DECODER_OUTPUTS,
+                                ENCODER_INPUTS, ENCODER_OUTPUTS)
+from tools.utils.get_config import get_config
+
+OPENVINO_DIR = '/opt/intel/openvino'
+
+FEATURES_SHAPE = 1, 20, 175, 512
+HIDDEN_SHAPE = 1, 512
+CONTEXT_SHAPE = 1, 512
+OUTPUT_SHAPE = 1, 256
+
+
+class ONNXExporter:
```

Comment:
Why it's name is `ONNXExporter`? Why not just `Exporter` ?    
The name `Exporter` is not used in your code.

Reply:
Renamed class